pipeline/experiment.py

```diff
@@ -189,6 +200,18 @@ class SessionComment(dj.Manual):
     """
 
 
+@schema
+class SessionDetails(dj.Manual):
```

Comment:
not sure if we spent enough time on this during the group review.. it's pretty standalone for the moment though

Reply:
I do think we need a more descriptive name for this table, seems like all of the attribute is about weight and water intake of the animal for that session. 
But do you have specific concerns about the design?

Reply:
as discussed no major concerns for now; will confirm with group before proceeding further